app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <%= link_to t("rails_tut"), Settings.railstutorial %>
+    by <%= link_to "Micheal Hartl", Settings.michealhartl %>
```

Comment:
I18n toàn bộ file này

Reply:
Anh ơi cái tên em không I18n đc, những cái còn lại I18n rồi ạ